turftopic/encoders/cohere.py

```diff
@@ -1,20 +1,10 @@
-import itertools
 import os
-from typing import Iterable, List
+from typing import Iterable
 
 import numpy as np
 
 from turftopic.encoders.base import ExternalEncoder
-
-
-def batched(iterable, n: int) -> Iterable[List[str]]:
-    "Batch data into tuples of length n. The last batch may be shorter."
-    # batched('ABCDEFG', 3) --> ABC DEF G
-    if n < 1:
-        raise ValueError("n must be at least one")
-    it = iter(iterable)
-    while batch := list(itertools.islice(it, n)):
-        yield batch
+from turftopic.encoder.utils import batched
```

Comment:
This has a typo. Also I would move the batched function to just `turftopic.utils` since it's not necessarily specific to encoders.